docs/startup/releases/versioned/release-notes.md

```diff
@@ -42,8 +43,12 @@ title: Release Notes
 
 ## Release Notes
 <!-- Add notes on new features, modified behavior of existing features, operational/performance improvements, new tools etc -->
-
-<TBD>
+* [SAMZA-1510](https://issues.apache.org/jira/browse/SAMZA-1510) - Samza SQL
+* [SAMZA-1438](https://issues.apache.org/jira/browse/SAMZA-1438) - Event-hub connectors for Samza
```

Comment:
Can we also include : 

Checkpoint provider for Azure table? (SAMZA-1486)

Reply:
Added.